src/main/scala/org/apache/spark/streamdm/streams/FileReader.scala

```diff
@@ -34,29 +34,32 @@ import org.apache.spark.streamdm.core.specification._
 import org.apache.spark.streamdm.streams.generators.Generator
 
 /**
- * FileReader is used to read data from one file of full data to simulate a stream data.
- *
- * <p>It uses the following options:
- * <ul>
- *  <li> Chunk size (<b>-k</b>)
- *  <li> Slide duration in milliseconds (<b>-d</b>)
- *  <li> Type of the instance to use, it should be "dense" or "sparse" (<b>-t</b>)
- *  <li> Data File Name (<b>-f</b>)
- *  <li> Data Header Format,uses weka's arff as default.(<b>-h</b>)
- * </ul>
- */
+  * FileReader is used to read data from one file of full data to simulate a stream data.
+  *
+  * <p>It uses the following options:
+  * <ul>
+  *  <li> Chunk size (<b>-k</b>)
+  *  <li> Slide duration in milliseconds (<b>-d</b>)
+  *  <li> Type of the instance to use, it should be "dense" or "sparse" (<b>-t</b>)
+  *  <li> Data File Name (<b>-f</b>)
+  *  <li> Data Header Format,uses weka's arff as default.(<b>-h</b>)
+  * </ul>
+  */
 
 class FileReader extends StreamReader with Logging {
 
   val chunkSizeOption: IntOption = new IntOption("chunkSize", 'k',
     "Chunk Size", 10000, 1, Integer.MAX_VALUE)
 
   val slideDurationOption: IntOption = new IntOption("slideDuration", 'd',
-    "Slide Duration in milliseconds", 1000, 1, Integer.MAX_VALUE)
+    "Slide Duration in milliseconds", 100, 1, Integer.MAX_VALUE)
 
   val instanceOption: StringOption = new StringOption("instanceType", 't',
     "Type of the instance to use", "dense")
 
+  val instanceLimitOption: IntOption = new IntOption("instanceLimit", 'i',
+    "Limit of number of instance", 100000,1, Integer.MAX_VALUE)

```

Comment:
100000, 1,
Without a blank between 100000 and 1.

Reply:
I added the space as you asked.